lib/src/base.dart

```diff
@@ -659,7 +659,7 @@ class Device {
   final String screenResolution;
 
   /// A floating point denoting the screen density.
-  final String screenDensity;
+  final double screenDensity;
```

Comment:
This is a breaking change. Let's do a major version bump.

Reply:
That's true. Should I bump it as part of this PR?

Reply:
Yep.

Reply:
Bumped the major and added changelog.